iocore/net/P_SNIActionPerformer.h

```diff
@@ -37,7 +37,7 @@
 #include "P_SSLNextProtocolAccept.h"
 #include "tscore/ink_inet.h"
 
-extern Map<int, SSLNextProtocolSet *> snpsMap;
+extern std::unordered_map<int, SSLNextProtocolSet *> snpsMap;
```

Comment:
Why don't you need #include <unordered_map> ?

Reply:
It's included by some other file that this file includes. I'll add it anyway.

Reply:
Yes. Never depend on "some other file" including what you need. Explicitly include what you need.